books-call-number/nodes/js/buildQuery.js

```diff
@@ -2,10 +2,12 @@ if (logLevel === 'DEBUG') {
   print('\nlogLevel = ' + logLevel + '\n');
   print('\ncall number start range = ' + startRange + '\n');
   print('\ncall number end range = ' + endRange + '\n');
+  print('\nlocationName = ' + locationName + '\n');
 }
-
 var where = 'TRUE';
 
+var locationNameArray = JSON.parse(locationName);
+
 if (startRange) {
```

Comment:
```suggestion
if (!!startRange) {
```

Being that this is JavaScript, we do not want to rely of the boolean check on the variable.
Instead, use a double exclamation, like `!!`.

Please do the same for the `if (endRange) {` below.